parasite_hub.py

```diff
@@ -0,0 +1,64 @@
+#PARASITE HUB
+#Description: This function calculates the parasite drag area
+#             of the hub, assuming it is a circular section cylinder
+#
+#References:-"Lezioni Aerodinamica dell'ala rotante", v2.05, Renato Tognaccini,
+#             section 7.5, "Potenza parassita della fusoliera"
+#           -"Fluid-Dynamic Drag", Hoerner, chapter 3, section 5, "Drag of round bodies"
+#
+#Author: Arianna Palumbo
+#Rotary Wing Aerodynamics course, prof. Renato Tognaccini
+#University of Naples Federico II
+#Academic Year 2023-2024
+#Date:14/07/2024
+#
+#Version:1.0.0
+
+
+
+import math as ma
+
+    def hub(self):
+        '''
+        This function calculates the parasite drag area of the hub, assuming it as a circular section
+        cylinder.
+        
+        Input:
+        -R:hub radius
+        -Lcs: hub height
+        -alpha: angle of attack
+        -S: reference area
```

Comment:
Seems the input is wrong.
It looks like you only have 1 input variable you named "self".